src/workflow_app/workflow/daemon.py

```diff
@@ -135,6 +135,24 @@ def restart(self):
         self.stop()
         self.start()
 
+    def status(self):
```

Comment:
Doesn't psutil do most of this with different interface?

Reply:
I could replace some of this with `psutil.pid_exists`